Makefile

```diff
@@ -155,7 +155,7 @@ BOOT_PART=$(INSTALLER)/boot
 BSP_IMX_PART=$(INSTALLER)/bsp-imx
 
 SBOM=$(ROOTFS).spdx.json
-
+COLLECTED_SOURCES=$(DIST)/collected_sources.tar.gz
```

Comment:
I think this should be `$(BUILD_DIR)/collected_sources.tar.gz` and not `$(DIST)`. We want it to end up in the specific build subdir:

* `$(DIST)` = `./dist/x86_64`
* `$(BUILD_DIR)` = `$(DIST)/<version>` = (e.g.) `./dist/x86_64/v3.2.1-abcd4566/`